x-pack/test/cases_api_integration/security_and_spaces/tests/common/index.ts

```diff
@@ -50,6 +49,8 @@ export default ({ loadTestFile }: FtrProviderContext): void => {
     loadTestFile(require.resolve('./internal/bulk_get_attachments'));
     loadTestFile(require.resolve('./internal/get_connectors'));
     loadTestFile(require.resolve('./internal/user_actions_get_users'));
+    loadTestFile(require.resolve('./internal/get_user_action_stats'));
```

Comment:
Just moved this into `internal` since that's where the rest of the tests for internal APIs are.